demo/bandwidth/src/bandwidth.js

```diff
@@ -0,0 +1,219 @@
+const { dia, g: geometry, V: vectorizer } = joint;
+
+const y = 200;
+const x1 = 100;
+const x2 = 700;
+
+const graph = new dia.Graph();
+const paper = new dia.Paper({
+    el: document.getElementById('paper'),
+    width: 800,
+    height: 300,
+    model: graph,
+    async: true,
+    sorting: dia.Paper.sorting.APPROX,
+    background: { color:  '#F3F7F6' },
+    defaultConnectionPoint: {
+        name: 'boundary',
+    },
+    restrictTranslate: (elementView) => {
+        // Restrict the element movement along the line only
+        const { height } = elementView.model.size();
+        return new geometry.Rect(x1, y - height, x2 - x1, 0);
+    }
+});
+
+paper.el.style.border = '1px solid #E5E5E5';
+
+// Draw a line behind all cells into the paper
+const line = vectorizer('line', {
```

Comment:
What is the thought process for creating the line on the 'Back' layer? Is it just to demonstrate the feature, or I guess you can separate the interactivity in this way maybe?

Would an equivalent be creating a line on the 'Cells' layer with a lower z value? Or am I missing some other differences?

Reply:
The `line` is not a cell (it's an SVGLine). I didn't want the line to be part of the graph. It's only a background.